mods/ts/rules/gdi-support.yaml

```diff
@@ -41,6 +41,129 @@ GAGATE_B:
 	Tooltip:
 		Name: GDI Gate
 
+GAFSDF:
+	Inherits@WALL: ^Wall
+	Inherits@EMPDISABLE: ^EmpDisable
+	Inherits@IDISABLED: ^DisabledOverlay
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 20
+		Prerequisites: gafire, ~structures.gdi
+		Description: Destroys units and blocks enemy fire when active.
+	-Building:
+	EnergyWall:
+		Dimensions: 1,1
+		Footprint: x
+		BuildSounds: place2.aud
+		Adjacent: 7
+		TerrainTypes: Clear, Rough, Road, DirtRoad, Green, Sand, Pavement
+		ActiveCondition: firestorm-active && !disabled
+		Weapon: Firestorm
+	SoundOnDamageTransition:
+		DamagedSounds:
+		DestroyedSounds:
+	Valued:
+		Cost: 50
+	CustomSellValue:
+		Value: 0
+	Tooltip:
+		Name: Firestorm Wall Section
+	Health:
+		HP: 200
+	Armor:
+		Type: Concrete
+	BlocksProjectiles:
+		RequiresCondition: firestorm-active && !disabled
+		Height: 9999
+	RequiresPower:
+	Power:
+		Amount: -2
+	RevealsShroud:
+		Range: 2c0
+	-Crushable:
+	LineBuild:
+		NodeTypes: firestormbarrier
+	LineBuildNode:
+		Types: firestormbarrier
+	GrantConditionOnLineBuildNode:
+		Type: firestormbarrier
+		Condition: firestormnode
+	WithWallSpriteBody:
+		Type: firestormbarrier
+	WithWallSpriteBody@ACTIVE:
+		Type: firestormbarrier
+		Sequence: enabled
+		RequiresCondition: firestorm-active && !disabled
+	WithIdleOverlay@FLASH:
+		Sequence: flash
+		RequiresCondition: firestorm-active && !disabled && firestormnode
+	WithRandomIntervalOverlay@BURST:
+		Sequence: burst
+		Palette: effect
+		IsPlayerPalette: false
+		RequiresCondition: firestorm-active && !disabled && firestormnode
+		MinRandomWaitTicks: 60
+		MaxRandomWaitTicks: 120
+	DamageMultiplier@INVULNERABLE:
+		RequiresCondition: firestorm-active
+		Modifier: 0
+	ExternalCondition@ACTIVE:
+		Condition: firestorm-active
+
+GAFIRE:
+	Inherits@BUILDING: ^Building
+	Inherits@IDISABLED: ^DisabledOverlay
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Firestorm Generator
+	Buildable:
+		BuildPaletteOrder: 80
+		Prerequisites: gatech, ~structures.gdi
+		Queue: Building
```

Comment:
Superweapon structures go in the Support tab.

Reply:
Yes, I also thought about that. However it was already quite crowded and `GAPLUG` is also in the `Building` queue.

Reply:
`GAPLUG` is a tech structure, while this is a support power / superweapon.

Reply:
Changed.